Formula/spotifyd.rb

```diff
@@ -20,26 +20,25 @@ class Spotifyd < Formula
   depends_on "pkg-config" => :build
   depends_on "rust" => :build
   depends_on "dbus"
-
-  on_linux do
-    depends_on "alsa-lib"
-  end
+  depends_on "portaudio"
 
   def install
     ENV["COREAUDIO_SDK_PATH"] = MacOS.sdk_path_if_needed
     system "cargo", "install", "--no-default-features",
-                               "--features=dbus_keyring,rodio_backend",
+                               "--features=dbus_keyring,portaudio_backend",
                                *std_cargo_args
   end
 
   def caveats
     <<~EOS
       Configure spotifyd using these instructions:
-        https://github.com/Spotifyd/spotifyd#configuration-file
+        https://spotifyd.github.io/spotifyd/config/File.html
+
+      For a minimal config, only "username" and one of the password fields are required.
```

Comment:
This should be in the upstream documentation

Reply:
This is not globally true for the upstream, it depends on the backend that is built by this Formula and on the command line flags we use. Anyway, no strong opinion if you want me to drop it.

Reply:
Yes, let's just get rid of the `caveats` entirely, since they're not working as is.

Reply:
Done.